### PR TITLE
Add: stnkl.EverythingToolbar version 0.7.1

### PR DESCRIPTION
**Pull request by quhxl**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20846)

### CODE REVIEW

File: manifests/s/stnkl/EverythingToolbar/0.7.1/stnkl.EverythingToolbar.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: stnkl.EverythingToolbar
+PackageVersion: 0.7.1
```
Comment (OfficialEsco) [inline manifests/s/stnkl/EverythingToolbar/0.7.1/stnkl.EverythingToolbar.installer.yaml:5]:
```suggestion
PackageVersion: 0.7.1.0
```

File: manifests/s/stnkl/EverythingToolbar/0.7.1/stnkl.EverythingToolbar.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: stnkl.EverythingToolbar
+PackageVersion: 0.7.1
```
Comment (OfficialEsco) [inline manifests/s/stnkl/EverythingToolbar/0.7.1/stnkl.EverythingToolbar.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.7.1.0
```

File: manifests/s/stnkl/EverythingToolbar/0.7.1/stnkl.EverythingToolbar.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: stnkl.EverythingToolbar
+PackageVersion: 0.7.1
```
Comment (OfficialEsco) [inline manifests/s/stnkl/EverythingToolbar/0.7.1/stnkl.EverythingToolbar.yaml:5]:
```suggestion
PackageVersion: 0.7.1.0
```

File: manifests/s/stnkl/EverythingToolbar/0.7.1/stnkl.EverythingToolbar.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+PublisherUrl: https://github.com/stnkl/EverythingToolbar
+PublisherSupportUrl: https://github.com/stnkl/EverythingToolbar/issues
+#PrivacyUrl: 
+Author: Stephan Rumswinkel
+PackageName: EverythingToolbar
+PackageUrl: https://github.com/stnkl/EverythingToolbar
+License: MIT License
+LicenseUrl: https://github.com/stnkl/EverythingToolbar/blob/master/LICENSE
+#Copyright: 
+#CopyrightUrl:
```
Comment (OfficialEsco) [inline manifests/s/stnkl/EverythingToolbar/0.7.1/stnkl.EverythingToolbar.locale.en-US.yaml:17]:
```suggestion
License: MIT License
LicenseUrl: https://raw.githubusercontent.com/stnkl/EverythingToolbar/master/LICENSE
Copyright: Copyright (c) 2021 Stephan Rumswinkel
CopyrightUrl: https://raw.githubusercontent.com/stnkl/EverythingToolbar/master/LICENSE
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by quhxl** [comment]
@OfficialEsco Do you know something about `Validation-No-Executables`?

**Comment by OfficialEsco** [comment]
> @OfficialEsco Do you know something about `Validation-No-Executables`?

No idea

https://github.com/microsoft/winget-cli/blob/5898d09203409d08742d97f8eb54e249316695e8/doc/windows/package-manager/package/winget-validation.md

https://docs.microsoft.com/en-us/windows/package-manager/package/winget-validation#internal-labels

Neither does the pipeline it seems...

**Comment by palenshus** [comment]
@OfficialEsco it looks like there are no install-time dependencies, but there is a runtime dependency, which we allow. You can go ahead and approve it if you're happy with the manifest, thanks!

**Review by OfficialEsco — APPROVED** [review]

**Comment by denelon** [comment]
@OfficialEsco we're working on the update to the documentation, but it means we didn't find an executable for the installer to "run".

**Comment by OfficialEsco** [comment]
> @OfficialEsco we're working on the update to the documentation, but it means we didn't find an executable for the installer to "run".

I was going to say that as it makes sense, but then surely these would fail

```
7/14/2021 2:58:31 PM
Installer Availability and Hash Validation
Status: Started
Installer: https://github.com/stnkl/EverythingToolbar/releases/download/0.7.1/EverythingToolbar-0.7.1.msi

7/14/2021 2:58:32 PM
Installer https://github.com/stnkl/EverythingToolbar/releases/download/0.7.1/EverythingToolbar-0.7.1.msi
Hash validation succeeded

7/14/2021 2:58:34 PM
Status: Completed
Installer: https://github.com/stnkl/EverythingToolbar/releases/download/0.7.1/EverythingToolbar-0.7.1.msi
Result: Succeeded

7/14/2021 2:58:35 PM
Static Analysis
Status: Started
Installer: https://github.com/stnkl/EverythingToolbar/releases/download/0.7.1/EverythingToolbar-0.7.1.msi

7/14/2021 2:58:50 PM
Status: Completed
Installer https://github.com/stnkl/EverythingToolbar/releases/download/0.7.1/EverythingToolbar-0.7.1.msi 
Result: Succeeded

7/14/2021 2:58:52 PM

****************************************************************************************
Installer Validation
****************************************************************************************
Status: Completed
-----------------------------------------------------
Installer https://github.com/stnkl/EverythingToolbar/releases/download/0.7.1/EverythingToolbar-0.7.1.msi
Result: Validation Succeeded
-----------------------------------------------------
Result: Succeeded
```
🤷‍♂️

**Comment by denelon** [comment]
Lots of moving pieces, and active development. I'm not sure what happened exactly. I was just responding to the comment about the label. There may be some other subtle edge cases where the label gets set. It's new enough we didn't even have it in our document, so I had to do a bit of digging myself.

**Pull request merged**